examples/grpc/opentracing/pom.xml

```diff
@@ -0,0 +1,118 @@
+        <docker.image.version></docker.image.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.helidon.examples.grpc</groupId>
+            <artifactId>helidon-examples-grpc-common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.helidon.grpc</groupId>
+            <artifactId>helidon-grpc-server</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.helidon.grpc</groupId>
+            <artifactId>helidon-grpc-client</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.helidon.tracing</groupId>
+            <artifactId>helidon-tracing-zipkin</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-docker-resources</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.build.directory}/distribution/container</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>src/main/docker</directory>
+                                    <filtering>true</filtering>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.spotify</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <configuration>
+                    <serverId>${docker.server.id}</serverId>
+                    <registryUrl>${docker.registry.url}</registryUrl>
+                    <imageName>${docker.image.name}</imageName>
+                    <dockerDirectory>${project.build.directory}/distribution/container</dockerDirectory>
+                    <imageTags>
+                        <imageTag>${docker.image.version}</imageTag>
+                        <imageTag>latest</imageTag>
+                    </imageTags>
+                    <resources>
+                        <resource>
+                            <targetPath>/</targetPath>
+                            <directory>${project.build.directory}</directory>
+                            <include>${project.build.finalName}-fat.jar</include>
```

Comment:
What's the "-fat" suffix ? We don't do fat jars in Helidon and I don't see anything that builds a jar ending with "-far", what am I missing ?

Reply:
Apart from the fact that this example creates a gRPC server the majority of the way it works was copied almost verbatim from the `examples/webserver/opentracing/pom.xml` including the bit you are questioning. I can only assume that it builds some sort of fat jar to put into the Docker image.

Reply:
This example seems broken, there is nothing in the pom.xml that builds a `-fat.jar`.